site/gatsby-site/src/components/RelatedIncidentsArea.js

```diff
@@ -83,9 +83,10 @@ const RelatedIncidentsArea = ({
                 {val?.incident_id && (
                   <span className="text-lg">
                     <a
-                      href={`/${locale}/cite/${val.incident_id}`}
+                      href={`${locale == 'en' ? '' : '/' + locale}/cite/${val.incident_id}`}
```

Comment:
Is this because `LocalizedLink` doesn't handle the `target` prop?

There are 2 path localization utils that would make a more robust solution: `localizedPath` (built-in)  and `switchLocalizedPath` (util made by me) please use any of those.